speaker/main.go

```diff
@@ -198,11 +200,12 @@ type controllerConfig struct {
 
 	// For testing only, and will be removed in a future release.
```

Comment:
i know it's unrelated to this PR, but do you know something about this comment from 4 years ago? the issue was closed.